### DISCUSSION TITLE
Welcome to GSEApy Discussions!

### DISCUSSION THREAD

**Discussion by zqfang** [zqfang/GSEApy#113] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about (without submit issues).
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with your thoughts.

**Comment by oesterei** [comment]
Hello. Is there a way to have the gseapy.gsea command also return the reference signature defined during the function? It currently returns enrichment plots, heat maps, and a handy cvs file of statistics like ES, leading-edge items, etc. Thanks! oesterei@gmail.com

**Reply by zqfang** [reply]
Now, v0.14.0 will write gene signature output to disk. If working in a python console, simply access the signature by the .gmt attribute of the returned object.
